lib/experimental/class-wp-navigation-fallbacks-gutenberg.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * WP_Navigation_Fallbacks_Gutenberg class
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+
+require __DIR__ . '/class-wp-menu-conversion-gutenberg.php';
+
+/**
+ * Manages Fallback behavior for Navigation menus.
+ *
+ * @access public
+ */
+class WP_Navigation_Fallbacks_Gutenberg {
+
+	/**
+	 * Gets (and/or creates) an appropriate fallback Navigation Menu.
+	 *
+	 * @return WP_Post|null the fallback Navigation Post or null.
+	 */
+	public static function get_fallback_menu() {
+
+		$fallback_sequence = array(
+			'get_most_recently_published_navigation',
+			'create_classic_menu_fallback',
+			'create_default_fallback',
```

Comment:
Nit: I'm not super keen on saving function names in a string like this as it makes it harder to find where they are called using search.

Reply:
I agree with @scruffian that we should not abstract like this. If we want to help on "making the sequence of fallbacks more explicit" a comment works better.

Reply:
@draganescu @scruffian I've reverted this.